.travis.yml

```diff
@@ -13,10 +13,20 @@ addons:
       - ant
       - ant-optional
 
+# set up a small VuFind and load some records
 before_script:
   - if [ ! -d downloads ]; then mkdir downloads; fi;
-  - if [ ! -f downloads/vufind-4.0.zip ]; then wget http://downloads.sourceforge.net/vufind/vufind-4.0.zip?use_mirror=osdn -O downloads/vufind-4.0.zip; fi;
-  - unzip downloads/vufind-4.0.zip
+  - if [ ! -f downloads/vufind-4.0.zip ]; then wget -nv http://downloads.sourceforge.net/vufind/vufind-4.0.zip?use_mirror=osdn -O downloads/vufind-4.0.zip; fi;
+  - unzip -q downloads/vufind-4.0.zip
+  - cd vufind-4.0
+  - ./solr.sh start
+  - VUFIND_HOME=$(pwd) ./import-marc-auth.sh tests/data/authority/authorityauth.mrc marc_auth_ils.properties
```

Comment:
I'm a little confused about why we need to create this test instance now in order for the build to pass but didn't need to before. Are there tests that actually rely on this data? And does it matter what data we index? Do we really need to index two different sets of authority data? (I'm mainly asking to see if there are some steps we can eliminate here to keep things as simple as possible).

Also, would it be cleaner to `export VUFIND_HOME=$(pwd)` once before starting Solr, rather than setting the variable on each individual line? Or did you try that without success?

Reply:
On creating the test instance, there are two things:
1. In order to set up for step-through debugging, the recommended thing for Solr handlers to to build a Junit test that one can hook into.
   1. Plus, that sets us up for some test-time sanity-checking of the browse response in the future.
1. To actually get a result from the browse handler, we need to have build browse indexes after having loaded some records.
I don't have tests yet that actually provide that checking, but as long as I needed the setup for debugging thought best to lay the groundwork. The advantage of creating a file of browse-handler-specific auth records is that I can use those names in test records and maybe even confirm that expected number of records are returned. 

But again don't have tests worked out at this point, just trying to lay some groundwork. Could possibly add a couple.

I did experiment with `export VUFIND_HOME=$(pwd)` yesterday and that actually worked!

Reply:
Test data removed, can re-add when we have tests.